lib/plucky/query.rb

```diff
@@ -64,6 +64,17 @@ def paginate(opts={})
         Pagination::Collection.new(docs, paginator)
       end
 
+      def find_in_batches(opts={})
+        batch_size = opts[:batch_size] || 1000
+        if block_given?
+          find_each(opts).each_slice(batch_size) do |documents|
```

Comment:
Ah, it's not intended. 
I believe `opts[:batch_size]` and an argument of `each_slice` should be same. 
Fixed 👍 d1aba20